heat-core-utils/pom.xml

```diff
@@ -34,6 +34,12 @@
             <artifactId>testng</artifactId>
             <version>6.8.21</version>
         </dependency>
+        <dependency>
+            <groupId>SnakeYAML</groupId>
+            <artifactId>SnakeYAML</artifactId>
+            <version>1.3</version>
```

Comment:
@mbarsocchi please, could you verify this dependency? https://mvnrepository.com/artifact/org.yaml/snakeyaml :-)